SwiftWisdomTests/StandardLibrary/Date/TimeOfDayTests.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  TimeOfDayTests.swift
+//  SwiftWisdom
+//
+//  Created by Patrick Butkiewicz on 3/1/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
+import Foundation
+import XCTest
+import SwiftWisdom
+
+final class TimeOfDayTests: XCTestCase {
```

Comment:
these tests will fail when run on non EST machine. Either inject the date formatter, or query the current timezone in the test and calculate based on offsets.